tests/storage/hostdisk.go

```diff
@@ -0,0 +1,27 @@
+package storage
```

Comment:
@orenc1 Can you please add a GPL license header? (w/o the year)

Reply:
done